src/Dapr.Actors/Builder/ActorProxyGeneratorBuilder.cs

```diff
@@ -236,14 +235,6 @@ public ActorProxyGeneratorBuilder(ICodeBuilder codeBuilder)
             }
         }
 
-        protected ActorProxyGenerator CreateProxyGenerator(
-            Type proxyInterfaceType,
-            IDictionary<InterfaceDescription, MethodBodyTypesBuildResult> methodBodyTypesResultsMap,
```

Comment:
this parameter is unused, which makes it a duplicate of the other overload (which it calls).